### ISSUE TITLE
AP calculation

### ISSUE THREAD

**Issue by profdresser** [lipefl/swade-tools#159]
I honestly can't remember and don't see anything in the documentation, but I thought that, before v.10, Tools would factor AP into damage when calculating Wounds. Was I misremembering? If not, then it doesn't seem to taking it into account now.
![Screenshot 2022-10-02 005450](https://user-images.githubusercontent.com/88245061/193440212-6f3951de-b9e1-44b7-9929-a97a566f6ffa.png)

**Comment by lipefl** [comment]
Yes. It takes into account. The problem is that it was relying on a swade system function that doesnt seem to be working. I made a little fix to work until it gets fixed. I'll open an issue there.
Please, update to v 1.10.3 
Thanks
